mne/report.py

```diff
@@ -1556,6 +1574,50 @@ def _render_forward(self, fwd_fname):
                                         repr=repr_fwd)
         return html
 
+    def _render_sensitivity_map(self, fwd_fname, subjects_dir):
+        """Render Sensitivity plots.
+        """
+        caption = u'Sensitivity : %s' % fwd_fname
+        div_klass = 'sensitivity'
+        img_klass = 'Sensitivity'
+        show = True
+
+        fwd = read_forward_solution(fwd_fname, surf_ori=True)
+        mag_map = sensitivity_map(fwd, ch_type='grad', mode='fixed')
+        mag_map.plot(time_label='Gradiometer sensitivity',
+                 subjects_dir=subjects_dir,
+                 clim=dict(lims=[0, 50, 100]))
+
+        html = []
+        from mayavi import mlab
+        global_id = self._get_id()
+        img = mlab.gcf()
+        img = _fig_to_img(fig=img)
+        html.append(image_template.substitute(img=img, id=global_id,
+                                              image_format='png',
+                                              div_klass=div_klass,
+                                              img_klass=img_klass,
+                                              caption=caption,
+                                              show=True))
+        import matplotlib.pyplot as plt
+        plt.figure()
+        plt.hist(mag_map.data.ravel(),
+                 bins=20, label=['Magnetometers'],
+                 color=['b'])
```

Comment:
this is a decision has some weight. Why do we show magnetometers by default? Some room for improvement at least.